### DISCUSSION TITLE
DCO for contributors

### DISCUSSION THREAD

**Discussion by fgreinacher** [renovatebot/renovate#14295] [Suggest an Idea]
Some folks (mostly employees of bigger companies) have problems signing CLAs. A more light-weight and usually legally "easier" alternative is the Developer Certificate of Origin (DCO). https://opensource.com/article/18/3/cla-vs-dco-whats-difference provides a neutral comparison.

Did you ever consider allowing a DCO in addition to (or instead of) the current CLA approach?

**Comment by rarkins** [comment]
We need the rights to reuse the code in some closed source uses such as the hosted app we run and I don't think any DCO grants that. We don't insist on "exclusive" rights to the code nor do we ask for a patent grant - both of which are sometimes present in CLAs, and about which I can understand some lawyers having concerns about. Speaking of lawyers at big companies, this project's CLA approach was modeled after Microsoft's and SAP's, and I think that "big companies" should be the best positioned to get good legal advice that what we ask for is reasonable.

Also FWIW opensource.com is a commercial website of Red Hat and they are by no means neutral in the "CLA debate" even if that article appeared to be. 

Here's another angle on the topic: https://writing.kemitchell.com/2018/01/06/CLAs-Are-Not-a-Sham.html
